examples/delaney/delaney_DAG.py

```diff
@@ -1,69 +0,0 @@
-"""
-Script that trains DAG models on delaney dataset.
-"""
-from __future__ import print_function
-from __future__ import division
-from __future__ import unicode_literals
-
-import numpy as np
-np.random.seed(123)
-import tensorflow as tf
-tf.set_random_seed(123)
-import deepchem as dc
-
-# Load Delaney dataset
-delaney_tasks, delaney_datasets, transformers = dc.molnet.load_delaney(
-    featurizer='GraphConv', split='random')
-train_dataset, valid_dataset, test_dataset = delaney_datasets
-
-# Fit models
-metric = dc.metrics.Metric(dc.metrics.pearson_r2_score, np.mean)
-
-max_atoms_train = max([mol.get_num_atoms() for mol in train_dataset.X])
-max_atoms_valid = max([mol.get_num_atoms() for mol in valid_dataset.X])
-max_atoms_test = max([mol.get_num_atoms() for mol in test_dataset.X])
-max_atoms = max([max_atoms_train, max_atoms_valid, max_atoms_test])
-
-transformer = dc.trans.DAGTransformer(max_atoms=max_atoms)
-train_dataset.reshard(512)
-train_dataset = transformer.transform(train_dataset)
-valid_dataset.reshard(512)
-valid_dataset = transformer.transform(valid_dataset)
-test_dataset.reshard(512)
-test_dataset = transformer.transform(test_dataset)
-# Do setup required for tf/keras models
-# Number of features on conv-mols
-n_feat = 75
-# Batch size of models
-batch_size = 64
-graph = dc.nn.SequentialDAGGraph(n_atom_feat=n_feat, max_atoms=max_atoms)
```

Comment:
Do we have a TensorGraph version of `SequentialDAGGraph` available? If not, we should add it on to #963.